### PR TITLE
Re-structed the Game HUD fixes #26

### LINKED ISSUE #26: game-hud structure
The hud structure seems all wrong.

Re layout the hud with

```
.game-hud
    .game-hud__score
    .game-hud__stats
    .game-hud__progress-bar
````

### PR DESCRIPTION
**Pull request by jordanabram1997**
Quickly updated the HUD structure ready for new set of updates

### CODE REVIEW

File: game.html [hunk 1]
```diff
@@ -26,42 +26,55 @@
...
-                <div class="dps">
-                  Punches:  <span class="game-hud__dps js-dps">0</span> / seconds
-                </div>
-                <div class="experience">
-                  Total Experience:  <span class="game-hud__experience js-experience">0</span>
-                </div>                      
-              </div>
-          </div>
-          <div class="progress">
+<!-- Game HUD -->
```
Comment (AaronLayton) [inline game.html]: Are these really needed? Adds bulk to the page HTML

File: game.html [hunk 2]
```diff
@@ -26,42 +26,55 @@
...
+      <div class="game-hud text-center">
```
Comment (AaronLayton) [inline game.html]: Can you fix the indentation across all the HTML

File: game.html [hunk 3]
```diff
@@ -26,42 +26,55 @@
...
+      	<div class="game-hud__score js-score">
+      		0
+      	</div>
+      	<div class="game-hud__stats">
+    	    <div class="game-hud__ppc">
```
Comment (AaronLayton) [inline game.html]: Do we need 3 unique class names for the `stat`? Can we change this to `.game-hud__stat` and then if we need to further style we can add an extra class on after

File: game.html [hunk 4]
```diff
@@ -26,42 +26,55 @@
...
+    	    	Punches: 
+    	      <span class="js-dpc">0</span> / click
+            </div>
+            <div class="game-hud__pps">
+              Punches:  
+              <span class="js-dps">0</span> / seconds
```
Comment (AaronLayton) [inline game.html]: Should be `second` - not plural

Comment (AaronLayton) [inline game.html]: `.js-dps` should be changed to `.js-pps`, remember to change in the JavaScript aswell

File: game.html [hunk 5]
```diff
@@ -26,42 +26,55 @@
...
-          <div class="progress">
+<!-- Game HUD -->
+      <div class="game-hud text-center">
+      	<div class="game-hud__score js-score">
+      		0
+      	</div>
+      	<div class="game-hud__stats">
+    	    <div class="game-hud__ppc">
+    	    	Punches: 
+    	      <span class="js-dpc">0</span> / click
```
Comment (AaronLayton) [inline game.html]: `.js-dpc` should be changed to `.js-ppc`, remember to change in the JavaScript aswell

File: game.html [hunk 6]
```diff
@@ -26,42 +26,55 @@
...
+              Punches:  
+              <span class="js-dps">0</span> / seconds
+            </div>
+	        <div class="game-hud__experience">
+	          Total Experience:  
+	          <span class="js-experience">0</span>
+	        </div>                      
+      	</div>
+      	<div class="game-hud__progress">
+			<div class="progress">
```
Comment (AaronLayton) [inline game.html]: Probably don't need this extra DIV `.progress`

File: game.html [hunk 7]
```diff
@@ -26,42 +26,55 @@
...
         <img src="img/penguin.png" alt="Punch me" />
       </div>
+<!-- CTA Button -->
       <div class="button-wrapper">
         <button class="btn btn-primary btn-raised js-clicker">Punch It!</button>
       </div>
+<!-- CTA Button End -->
     </div>
+
+
```
Comment (AaronLayton) [inline game.html]: Lets remove this last blank line - make it just the 1 blank line

File: game.html (hunk 1 above)
Comment (jordanabram1997) [inline game.html]: okay Aaron thank you I will sort this immediately.

File: js/game.js [hunk 8]
```diff
@@ -58,8 +58,8 @@ function loadData(){
 
 function setEverything(){
 	$(".js-score").html(numberWithCommas(Math.floor(settings.score))); 
-	$(".js-dpc").html(numberWithCommas(settings.dpc));
-	$(".js-dps").html(numberWithCommas(settings.dps));
+	$(".js-ppc").html(numberWithCommas(settings.dpc));
```
Comment (AaronLayton) [inline js/game.js]: Can you change the `settings.dpc` & `settings.dps` aswell

Comment (AaronLayton) [inline js/game.js]: Make sure to change all references of DPC and DPS

File: game.html [hunk 9]
```diff
@@ -1,97 +1,96 @@
...
-              <div>
-                <div class="dpc">
-                  Punches: <span class="game-hud__dpc js-dpc">0</span> / click
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="mobile-web-app-capable" content="yes">
+        <title>Penguin Puncher</title>
+        <!-- Bootstrap -->
```
Comment (AaronLayton) [inline game.html]: We are missing the new line between the title and Bootstrap comment, can we add it again for clarity

File: game.html [hunk 10]
```diff
@@ -1,97 +1,96 @@
...
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <link rel="manifest" href="manifest.json">
+        <link href="css/game.css" rel="stylesheet">
+        <link rel="stylesheet" href="css/animate.min.css">
+        <link href='http://fonts.googleapis.com/css?family=Sigmar+One' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/0.2.2/css/material-wfont.min.css">
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/0.2.2/css/material.min.css">
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/0.2.2/css/ripples.min.css">
+        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
```
Comment (AaronLayton) [inline game.html]: Same newline here please

File: game.html [hunk 11]
```diff
@@ -1,97 +1,96 @@
...
-                </div>                      
-              </div>
-          </div>
-          <div class="progress">
-              <div class="progress-bar progress-bar-striped active js-progress" style="width: 0%">
-                <div class="text-center">
-                  Level:  <span class="game-hud__level js-level">0</span> 
+                <div class="game-hud__progress">
+                    <div class="progress-bar progress-bar-striped active js-progress" style="width: 0%">
+                        <div class="text-center">
```
Comment (AaronLayton) [inline game.html]: Remove the extra DIV with `.text-center` - add that class to the `.progress-bar`

File: game.html [hunk 12]
```diff
@@ -1,97 +1,96 @@
...
+                            Level:  <span class="game-hud__level js-level">0</span> 
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="container-upgrade">
+                <div class="game-container__upgrades output">
```
Comment (AaronLayton) [inline game.html]: Do we need `.output`?

Comment (jordanabram1997) [inline game.html]: I been reviewing where `.output` is used I have found it with `game.js` and is used to setup the Upgrades. In the `game.js` file `$(".output").html(html);` is on line 79 is this used to set the language that the browser needs to read?

Comment (AaronLayton) [inline game.html]: Ok lets change this and all references to something like `.js-updates-output` ?

File: game.html [hunk 13]
```diff
@@ -1,97 +1,100 @@
...
-        <div class="game-container__clicker">
-          <div class="game-hud text-center">
-              <div class="game-hud__score js-score">0</div>
-              <div>
-                <div class="dpc">
-                  Punches: <span class="game-hud__dpc js-dpc">0</span> / click
+        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+        <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
```
Comment (AaronLayton) [inline game.html]: Can you tab these back in

File: game.html [hunk 14]
```diff
@@ -1,97 +1,100 @@
...
+        <script id="upgrade-template" type="text/x-handlebars-template">
+            {{#each upgrades}}
+            <div class="upgrade-item media" data-id="{{@index}}">
+                <div class="pull-left">
+                    <img class="media-object" src="{{image}}" alt="{{title}}">
+                </div>
+                <div class="media-body">
+                    <div class="pull-right upgrade-item__price">{{addCommas price}}</div>
+                    <h4 class="media-heading">{{title}}</h4>
+                    DPC: {{addCommas dpc}} DPS: {{addCommas dps}}
```
Comment (AaronLayton) [inline game.html]: DPC & DPS still uses

**Comment by AaronLayton** [comment]
Ok cool - can you squash your commits and I will pull

**Comment by AaronLayton** [comment]
Looks good

**Pull request merged**